### ISSUE TITLE
Unified Alerting and OpenTSDB timestamp issue

### ISSUE THREAD

**Issue by NikolaBorisov** [grafana/grafana#36245]
**What happened**:
Using Grafana 8.0.3 and ngalert enabled, and OpenTSDB 2.4 as data source I tried to create an Alert. It looks like the alert screens is not handling the OpenTSDB timeseries seconds/milliseconds. So the same query that works fine in Graph panel on the Create Alert rule page shows the events with timestamps of 1970. 

**What you expected to happen**:
I was expecting to see exactly the same graph on the Create Alert rule page as on the graph for which I'm creating the alert.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.0.3
- Data source type & version: OpenTSDB 2.4
- OS Grafana is installed on: Linux
- User OS & Browser:
- Grafana plugins: ngalert enabled
- Others:

**Comment by NikolaBorisov** [comment]
@zuchka Here is the new issue I filed after #36191

**Comment by idafurjes** [comment]
@NikolaBorisov could you please share the query and some screenshots with the incorrect timestamp

**Comment by NikolaBorisov** [comment]
![bug](https://user-images.githubusercontent.com/184322/125691386-80f618cc-f7cb-46a8-966f-24111b8a96f6.png)
This is on the Create Alert Rule page

**Comment by NikolaBorisov** [comment]
Note how the timestamps are 1970, but the data is from the last 90 days

**Comment by NikolaBorisov** [comment]
@idafurjes Any chance you can take a look?

**Comment by NikolaBorisov** [comment]
@idafurjes This one seems to be fixed, but I ran into another issue : https://github.com/grafana/grafana/issues/38729
Maybe you can take a look.
